### DISCUSSION TITLE
Will pyright add support for the draft PEP 698 (typing.override)?

### DISCUSSION THREAD

**Discussion by tmke8** [microsoft/pyright#4046] [General]
It's of course only a draft right now, but pyright has often been very quick with such things so I thought I'd ask :smile: 

`typing-extensions` has runtime support for it now: https://github.com/python/typing_extensions/blob/main/CHANGELOG.md#release-440-october-6-2022

**Comment by erictraut** [comment]
This will be included in the next release of pyright.
